e2e/tests/solicitorCreatesClaim_test.js

```diff
@@ -12,15 +12,15 @@ Scenario('Solicitor creates claim @create-claim', async (I) => {
 
 Scenario('Solicitor confirms service', async (I) => {
   await I.confirmService();
-  await I.see('updated with event: Confirm service');
+  await I.waitForElement(locate('exui-alert').withText('updated with event: Confirm service'));
 });
 
 Scenario('Solicitor requests extension', async (I) => {
   await I.requestExtension();
-  await I.see('updated with event: Request extension');
+  await I.waitForElement(locate('exui-alert').withText('updated with event: Request extension'));
```

Comment:
Hmm you think just the wait is enough of an assertion for these tests? I guess they can’t pass without 🤷🏼‍♂️

Reply:
it does the work, but will change it to `seeElement` in another PR, as I'd like to get this merged to fix master